troposphere/static/js/components/admin/ImageRequest.react.js

```diff
@@ -136,141 +136,143 @@ export default React.createClass({
 
         return (
         <div className="request-admin pull-right admin-detail">
-            <div>
-                <h3>Request Information</h3>
+            <div style={{ marginBottom: "20px" }}>
+                <h3 className="t-body-2">Request Information</h3>
                 <div>
-                    Request ID:
-                    {request.get("id")}
+                   Request ID:
+                   {` ${request.get("id")}` }
```

Comment:
I think there is a little more value to template literals than that, but I do agree that moving the logic to assignment is much nicer. In this case there are a lot of them so I'm going to punt.

[Template Literal MDN Docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)